README.md

```diff
@@ -1,22 +1,22 @@
-# Todo Challenge
+[![Build Status](https://travis-ci.org/c-christenson/todo_challenge.svg?branch=master)](https://travis-ci.org/c-christenson/todo_challenge)
 
-* Deadline: submit completed pull request by 9am on Monday
-* You may use whatever level of JavaScript you feel comfortable with - pure JS, jQuery, Angular, or whatever weird and wonderful framework you want to try. Extra points for DogeScript
+###How to Run
+`git clone https://github.com/c-christenson/todo_challenge.git`  
+`cd todo_challenge`  
+`bundle install`  
```

Comment:
Instead of bundle install:
- npm install
- bower install
- npm install -g protractor
- webdriver-manager update --standalone